src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadTaxonomiesData.php

```diff
@@ -99,9 +99,10 @@ protected function createTaxonomy($code, array $taxonomyName, array $taxonsArray
                     $this->setReference('Sylius.Taxon.'.$taxonName, $taxon);
                 }
             }
-            $taxonomy->addTaxon($taxon);
+
+            $rootTaxon->addChild($taxon);
```

Comment:
This method only supports 2 levels of taxons hierarchy, I read the code one more time, it seems to me it's correct. I've tested fixtures and it seems to be working as expected.